src/app/wizard/views/DataplaneUniversal.vue

```diff
@@ -464,282 +466,255 @@ networking:
-        )
-      }
-
-      return false
-    },
-  },
-  watch: {
-    'validate.univDataplaneId'(value) {
-      this.validate.univDataplaneId = kebabCase(value)
-    },
+function hideSiblings(): void {
+  // this triggers when to hide the siblings related to the Scanner
+  // component that need to be hidden once the scan succeeds.
+  hideScannerSiblings.value = true
+}
 
-    'validate.univDataplaneServiceName'(value) {
-      const newName = kebabCase(value)
+async function scanForEntity(): Promise<void> {
+  const { meshName, univDataplaneId } = validate.value
 
-      this.validate.univDataplaneServiceName = newName
+  // reset things if the user is starting over
+  isComplete.value = false
+  scanError.value = false
 
-      if (this.validate.univDataplaneServiceName === '') {
-        this.validate.univDataplaneId = ''
-      } else {
-        this.validate.univDataplaneId = kebabCase(`${value}-${this.randString}`)
-      }
-    },
+  // do nothing if there is no Mesh nor Dataplane found
+  if (!meshName || !univDataplaneId) return
 
-    'validate.univDataplaneNetworkServicePort'(value) {
-      const newId = value.replace(/[a-zA-Z]*$/g, '').trim()
+  try {
+    const response = await kumaApi.getDataplaneFromMesh({ mesh: meshName, name: univDataplaneId })
 
-      this.validate.univDataplaneNetworkServicePort = newId
-    },
+    if (response.name?.length > 0) {
+      scanFound.value = true
+    } else {
+      scanError.value = true
+    }
+  } catch (err) {
+    scanError.value = true
 
-    'validate.univDataplaneNetworkDPPort'(value) {
-      const newId = value.replace(/[a-zA-Z]*$/g, '').trim()
+    console.error(err)
+  } finally {
+    isComplete.value = true
+  }
+}
 
-      this.validate.univDataplaneNetworkDPPort = newId
-    },
-  },
-  methods: {
-    hideSiblings() {
-      // this triggers when to hide the siblings related to the Scanner
-      // component that need to be hidden once the scan succeeds.
-      this.hideScannerSiblings = true
-    },
-    scanForEntity() {
-      const { meshName, univDataplaneId } = this.validate
-
-      // reset things if the user is starting over
-      this.scanComplete = false
-      this.scanError = false
-
-      // do nothing if there is no Mesh nor Dataplane found
-      if (!meshName || !univDataplaneId) return
-
-      kumaApi.getDataplaneFromMesh({ mesh: meshName, name: univDataplaneId })
-        .then((response) => {
-          if (response?.name?.length > 0) {
-            this.isRunning = true
-            this.scanFound = true
-          } else {
-            this.scanError = true
-          }
-        })
-        .catch((error) => {
-          this.scanError = true
-
-          console.error(error)
-        })
-        .finally(() => {
-          this.scanComplete = true
-        })
-    },
-    compeleteDataPlaneSetup() {
-      this.$store.dispatch('updateSelectedMesh', this.validate.meshName)
+function compeleteDataPlaneSetup(): void {
```

Comment:
Similar to the other place I saw this, just a spelling fix up

Reply:
Done.